libs/design/breadcrumb/src/breadcrumb/breadcrumb.component.ts

```diff
@@ -24,9 +29,35 @@ import {
   ],
   changeDetection: ChangeDetectionStrategy.OnPush,
   encapsulation: ViewEncapsulation.None,
-  standalone: true,
 })
 
-export class DaffBreadcrumbComponent {
+export class DaffBreadcrumbComponent implements AfterContentInit {
+  /**
+   * @docs-private
+   */
   @HostBinding('class.daff-breadcrumb') class = true;
+
+  /**
+   * @docs-private
+   */
+  @ContentChildren(DaffBreadcrumbItemDirective) breadcrumbItems!: QueryList<DaffBreadcrumbItemDirective>;
+
+  /**
+   * @docs-private
+   */
+  ngAfterContentInit() {
+    this.updateActiveState();
+    this.breadcrumbItems.changes.subscribe(() => this.updateActiveState());
```

Comment:
This needs to be takeUntilDestroyed to prevent memory leaks.

Reply:
updated